lib/server/services/lhcPeriod/LhcPeriodStatisticsService.js

```diff
@@ -66,7 +79,8 @@ class LhcPeriodStatisticsService {
             throw new BadParameterError('Can not find without LHC Period id or name');
         }
 
-        queryBuilder.includeAttribute(...DISTINCT_ENERGIES_IN_STATISTICS_ATTRIBUTE);
+        queryBuilder.includeAttribute(DISTINCT_ENERGIES_IN_STATISTICS_ATTRIBUTE);
+        queryBuilder.includeAttribute(PDP_BEAM_TYPES_IN_STATISTICS_ATTRIBUTE);
```

Comment:
I think here you should name the constant `LHC_PERIOD_STATISTICS_DISTINCT_ENERGIES_SUBQUERY` and put the alias here, constant only for the query itself

Reply:
applied